--- Makefile.PL.orig	2024-01-06 12:57:40.000000000 +1100
+++ Makefile.PL	2024-01-06 12:59:37.000000000 +1100
@@ -1,12 +1,14 @@
 use strict;
 use warnings;
 
-use Alien::Base::Wrapper 'WriteMakefile';
+#use Alien::Base::Wrapper 'WriteMakefile';
+use ExtUtils::MakeMaker;
+
 
 # See lib/ExtUtils/MakeMaker.pm for details of how to influence
 # the contents of the Makefile that is written.
 WriteMakefile(
-        alien_requires => { 'Alien::CFITSIO' => '4.3.0' },
+        #alien_requires => { 'Alien::CFITSIO' => '4.3.0' },
         NAME         => 'Astro::FITS::CFITSIO',
         VERSION_FROM => 'CFITSIO.pm',             # finds $VERSION
         OBJECT       => 'CFITSIO.o util.o',
